website/templates/base.mako

```diff
@@ -188,10 +188,13 @@
                     id: ${ user_id | sjson, n },
                     locale: ${ user_locale | sjson, n },
                     timezone: ${ user_timezone | sjson, n },
-                    entryPoint: ${ user_entry_point | sjson, n }
+                    entryPoint: ${ user_entry_point | sjson, n },
+                    emailsToAdd: ${ user_email_verifications | sjson, n }
                 },
                 popular: ${ popular_links_node | sjson, n },
-                newAndNoteworthy: ${ noteworthy_links_node | sjson, n }
+                newAndNoteworthy: ${ noteworthy_links_node | sjson, n },
+                confirmedEmailURL: ${ api_url_for('unconfirmed_email_add') | sjson, n }
```

Comment:
Minor: This probably isn't necessary, especially because there are no path variables in the URL. It's just `/api/v1//confirmed_emails/`.